include/dd/ComplexTable.hpp

```diff
@@ -365,6 +386,21 @@ namespace dd {
 
         [[nodiscard]] fp colRatio() const { return static_cast<fp>(collisions) / lookups; }
 
+        std::map<std::string, std::size_t> getStatistics() {
+            return {
+                    {"hits", hits},
+                    {"collisions", collisions},
+                    {"lookups", lookups},
+                    {"inserts", inserts},
+                    {"insert_collisions", insert_collisions},
+                    {"find_or_inserts", find_or_inserts},
+                    {"upper_neighbors", upper_neighbors},
+                    {"lower_neighbors", lower_neighbors},
```

Comment:
camelCase

Reply:
Not for the keys in the map (which is supposed to offer an easy way to expose the stats via json).

Reply:
But it's just as easy in camelCase as it is in snakeCase. Why not stick to one?

Reply:
okay, camelCase it is